src/app/pages/administration/jobs/scheduled/cron-gen.scss

```diff
@@ -9,20 +9,72 @@ $prefix: ".chpl";
     }
 
     #{$prefix}-col-2 {
-        width: calc(100% / 6);
+        width: calc(100% / 3);
```

Comment:
This one seems odd to me. I think the -col-2 in bootstrap is for things that "span 2 columns", which is why they make the width be 1/6 of the full width (as there are 12 columns, and spanning 2 would be 1/6th the width). By changing it to /3 does that double the width? So effectively making this span 4, instead of 2, columns?

Reply:
The only reason I change this was to fix stacking type in the "specific day". I can show you my local if needed.